runstatus.go

```diff
@@ -198,6 +207,8 @@ func (m *RunstatusMaintenance) ID() (int, error) {
 
 // UpdateRunstatusMaintenance update runstatus Maintenance
 func (client *Client) UpdateRunstatusMaintenance(ctx context.Context, page string, id int, event RunstatusEvent) error {
+	// XXX: GET /pages/$page | jq .maintenances_url
+	//      GEt maintenances_url + "/$id" | jq .events_url
```

Comment:
@pierre-emmanuelJ some ideas. :1st_place_medal:

Reply:
LGTM 💯 better to have good URL in reply instead of building them manually. It's safer and more HATE-OAS compliant 👍